### ISSUE TITLE
Last week's TW kernel seems to break MOK?? and render system not bootable?

### ISSUE THREAD

**Issue by esteelpaz** [geckolinux/geckolinux-project#358]
@geckolinux 

Not sure if you "know" about this problem but yesterday in TW had an issue with what would have been last weeks kernel apparently breaking "MOK" (according to a post on the forum) and system booted to error demsg . . . even after a 472 package upgrade.

In that case I found with trial and error that I could use "advanced options" in grub to go back to the prior kernel and then it booted OK . . . followed by "1673 packages to upgrade" . . . took a bunch of time to get it done.  The "new" kernel seems to fix the problem.

Problem was today is "Gecko" day and same problem with last weeks kernel booting to TTY error . . . but, in the case of Gecko, the Grub "advanced options" list just shows the same data for each line item and clicking on where "recovery" for the older kernel would be . . . failed.  I'll have to get the SG2 disk out to try to boot the partition, but no time for it today . . . jes giving you a head's up, seems like the 5.16.8 - 1 default kernel "fixes" the problem.

**Comment by esteelpaz** [comment]
And, today in another partition running Gecko . . . same problem . . . .  Seems like last weeks upgrades created a problem on reboot across the range of SUSE products.  I have Gecko on a newer machine that later today I'll check to see if it boots cleanly . . . haven't upgraded it for a couple of weeks???

**Comment by esteelpaz** [comment]
Booting over in my Sys76 linux laptop, Gecko had a moment . . . but then did boot via 15.6.8-1 kernel, but back on the Mac Pro when I went to boot the third SUSE distro, of Gecko MATE using SG2 could not boot the 15.6.8 kernel option, had to drop back to 15.5 . . . to then run zypper to install now 15.6.10 . . . .

And that is what is upgrading in my Gecko Plasma edition in the Gaz15 . . . .  

I'll have to boot the master GRUB system to run the Yast Bootloader, before I can check back into the Gecko's . . . .  If all goes well I won't bother to post back here . . . .  It now seems like 15.6.10 is the "fixer"????

**Comment by esteelpaz** [comment]
Issue of some sort continues in two Gecko partitions that booted to TTY, rather than GUI log in . . . didn't spend time trying to figure it out . . . .

**Comment by esteelpaz** [comment]
@geckolinux 

Spent about an hour trying to get to the Gecko rolling GUI this morning . . . did not prevail.  System seems to get to the tty login in the dmesg and then hangs on the IPV6 "uplink" lines??

I got to the TTY log in and ran a zypper, it showed something like 335 packages to upgrade, clicked go . . . got about half way through and then hung.  Had to restart the zypper operation 4 times to get all of the packages installed . . . .  Rebooted and went to TW Yast to update the grub bootloader for all of the 8 systems, shut down.

On cold boot went to the Gecko option and same deal, hangs up in the dmesg.  Tried "advanced options" and went to line two for the "older kernel" and in the dmesg it said, "failed to load kernel."  Went back to advanced options and selected the top line . . . everything runs "OK" until it gets to the IPV6 uplink line and then it hangs for several minutes . . . .

Something is "borked" . . . zypper doesn't seem to know what it is doing.  In the first round of zypper upgrades it hung up on installing the nvidia 470 xxxxx package.  When I control +d or +c out of it, when I ran zypper dup -l again it showed that it was "removing nvidia 470" . . . and then hung on the 3rd package out of 8 that were left to install . . . .

I rebooted and now I'm in Manjaro to type out this data.  I anticipate that when I get to my other Gecko install the same problems will continue over there . . . .
